x-pack/plugins/actions/server/plugin.ts

```diff
@@ -747,12 +748,13 @@ export function renderActionParameterTemplates<Params extends ActionTypeParams =
   actionTypeId: string,
   actionId: string,
   params: Params,
-  variables: Record<string, unknown>
+  variables: Record<string, unknown>,
+  logger: Logger
 ): Params {
   const actionType = actionTypeRegistry?.get(actionTypeId);
   if (actionType?.renderParameterTemplates) {
-    return actionType.renderParameterTemplates(params, variables, actionId) as Params;
+    return actionType.renderParameterTemplates(logger, params, variables, actionId) as Params;
```

Comment:
Why don't we use the logger in `actions` plugin but pass from the source plugin such as alerting?
To see which plugin caused the error?

Reply:
We can definitely do that!

Reply:
Updated in https://github.com/elastic/kibana/pull/176014/commits/58915d224e1364140eb4309dfef619c8e369b062